src/TrackPanel.cpp

```diff
@@ -735,11 +742,62 @@ void TrackPanel::OnMouseEvent(wxMouseEvent & event)
          }
       } );
    }
+    
+    if(event.RightDown())
```

Comment:
Context menu should happen at mouse up, not down, and on Windows should also happen for the special menu key.  See how LabelTrackView does it.